src/components/AssetSwiper/AssetSwiper.js

```diff
@@ -0,0 +1,56 @@
+// Import Swiper React components
+import React, { useState, useEffect } from "react";
+import { Typography, IconButton } from "@material-ui/core";
+import { Swiper, SwiperSlide } from "swiper/react";
+import AddIcon from "@material-ui/icons/Add";
+import CircularProgress from "@material-ui/core/CircularProgress";
+// Import Swiper styles
+import "swiper/swiper.scss";
+import AssetCard from "../AssetCard/AssetCard";
+import { Redirect } from "react-router-dom";
+import { useStyles } from "./Comps/AssetSwiper.style";
+import AssetSwiperMobile from "./Comps/AssetSwiperMobile";
+export default ({ list, title, profileSwiper, spinner }) => {
+  const classes = useStyles();
+  const [redirect, setRedirect] = useState(false);
+  const [renderList, setRenderList] = useState([]);
+
+  if (list.length > 0 || !spinner) {
```

Comment:
Logic is easier to read upside down ->

if (spinner) {
return loader
} else {
return list
}